src/TestSuite/IntegrationTestCase.php

```diff
@@ -219,6 +229,43 @@ public function cookie($name, $value)
     }
 
     /**
+     * Returns the encryption key to be used.
+     *
+     * @return string
+     */
+    protected function _getCookieEncryptionKey() {
+        if (isset($this->_cookieEncriptionKey)) {
+            return $this->_cookieEncriptionKey;
+        }
+        $key = Security::salt();
+        if (isset($key)) {
+            return $key;
+        }
+        $key = Security::salt('foo!foo!foo!foo!foo!foo!foo!foo!');
```

Comment:
If beginners forget `Security::salt('...')` before `cookieEncrypted()`, they will see `InvalidArgumentException: Invalid key for encrypt(), key must be at least 256 bits (32 bytes) long.` I think it is a little difficult for them to resolve it by themselves. If there is this default salt, they can `cookieEncrypted()` even if they forget to set the salt. It is the reason why I introduced this default salt. The random string is no meaning. I just wanted the 32 bytes string. How do you feel this? Should I remove this?